wsi/overlaybuffer.h

```diff
@@ -71,12 +71,14 @@ class OverlayBuffer {
 
   virtual const ResourceHandle& GetGpuResource() = 0;
 
-  // Returns Media resource for this buffer which can be used by compositor.
-  // Surface will be clipped to width, height even if buffer size is
-  // greater than these values.
+// Returns Media resource for this buffer which can be used by compositor.
+// Surface will be clipped to width, height even if buffer size is
+// greater than these values.
+#ifndef DISABLE_VA
```

Comment:
why comment this out ?

Reply:
this function is only used if VA enabled. and It cause compiation issue in drmbuffer.cpp if va is not included. VA_SURFACE_INVALID is not defined. Anyway it make no sense to keep this function if VA is not enabled

Reply:
Conceptually you could have different Media backend not using libva. If we are isolating libva changes, this is not right. We should have the implementation commented out with an error log saying "Function is not implemented" when libva is disabled. Also, enable this as one of the build step in travis ci.

Reply:
Done